app/src/components/card/cardImage.tsx

```diff
@@ -35,12 +37,14 @@ export const CardImage = ({ record, imageHeight }: CardImageProps) => {
         }}
         width={initialImageHeight * 2}
         height={initialImageHeight}
+        decoding="sync"
         onError={(_event) => {
           console.warn(
             `CardImage: Card image failed to load, fallback image loaded instead. ImageURL: ${record.imageURL}`
           );
           setImageSrc("/noImage.png");
         }}
+        placeholder="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mN8WQ8AAlcBas53/MIAAAAASUVORK5CYII="
```

Comment:
It's meant to be just a gray rectangle (this format is what Next wants for some reason). I don't want to use the `noImage` as the placeholder because it'll be confusing when the image does load